services/document_conversion/v1.js

```diff
@@ -86,17 +88,16 @@ module.exports = function(RED) {
 			
 				stream_payload(info.path, msg.payload, function (format) {
 					
-					// convert a document
+					node.status({fill:"blue", shape:"dot", text:"converting"});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2426928331/issues/source?bid=3225115&fileBranchId=3240515#l91)